packages/p2p/src/components/order-details/order-details-wrapper.jsx

```diff
@@ -26,11 +33,7 @@ const OrderDetailsWrapper = ({ children, onPageReturn, page_title }) => {
                     onClick={() => sendbird_store.setShouldShowChatModal(true)}
                 />
             )}
-            renderPageFooterChildren={() =>
-                order_store.order_information.should_show_order_footer && (
-                    <OrderDetailsFooter order_information={order_store.order_information} />
-                )
-            }
+            renderPageFooterChildren={shouldShowOrderDetailsFooter(order_store)}
```

Comment:
@adrienne-deriv should we be using `renderPageFooterChildren` like render Prop - https://reactjs.org/docs/render-props.html

Reply:
The `renderPageFooterChildren` PropTypes in the `MobileFullPageModal` component accepts only a function type 😢